src/Sylius/Component/Core/Model/TaxonImageInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Model;
+
+use Sylius\Component\Resource\Model\CodeAwareInterface;
+
+interface TaxonImageInterface extends ImageInterface, CodeAwareInterface
```

Comment:
Yup, but I'm not an author of this class and I guess that I shouldn't add someone else.